### ISSUE TITLE
CustomEvents not working

### ISSUE THREAD

**Issue by eddiemoore** [mysticatea/event-target-shim#1]
I'm trying to dispatch a custom event but currently it's not working.

dispatcher
```js
const evt = new CustomEvent('testevent', { detail: 'details' })
this.dispatchEvent(evt)
```
listener
```js
dispatcher.addEventListener('testevent', e => {
  console.log(e)
})
```

output from console.log
```js
{ type: 'testevent',
  target: { _uri: 'http://example.com' },
  currentTarget: { _uri: 'http://example.com' },
  eventPhase: 2,
  bubbles: false,
  cancelable: false,
  timeStamp: 1432690145315,
  isTrusted: false }
```

**Comment by mysticatea** [comment]
Thank you for the report.
I will check it this evening (JST).

**Comment by eddiemoore** [comment]
:+1: Thanks. I tried to compile the files in #2 but it didn't seem to be working properly. Was there a special order that you used to build the files?

**Comment by mysticatea** [comment]
The `detail` property exists in the prototype chain of the event wrapper object.
But I found that it throws `TypeError` when accessed the `CustomEvent.prototype.detail` getter on IE and FF :(

Your PR looks good to me, I will merge it.
Thanks!

Hm, there is just a command `npm run build` to build.

**Comment by mysticatea** [comment]
My added test (9b237e03) had been working, so I released `v1.0.5`.
Please try it!
